### PR TITLE
Ana/2nd governance qa

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Fixes for now:

- display Treasury beneficiary in ProposalDescription
- refactor ProposalDescription to receive `proposal`
- fix depositTx for Polkadot
- fix governanceOverview cards being asymmetric on PageProposals

Work in progress

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
This PR comes from #4953 , since I couldn't work with `base-node-subscription` for Polkadot

**Comment by faboweb** [comment]
🚀 Deployed on https://5f6892556d590fcdc4b7fa1b--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
Another thing to fix, percentage of votes when it is Deposit Period:

image

**Comment by Bitcoinera** [comment]
Deposits are gone... Fixing it

image

**Comment by Bitcoinera** [comment]
https://github.com/luniehq/lunie/pull/4949/files#r492061521

File: app/src/ActionModal/components/ModalDeposit.vue [hunk 1]
```diff
@@ -126,7 +126,8 @@ export default {
         amount: {
           amount: this.amount,
           denom: this.denom,
-        }
+        },
+        depositsSum: this.deposits.length,
```
Comment (faboweb) [inline app/src/ActionModal/components/ModalDeposit.vue:130]: this sounds like the sum of all deposits but is the count. Let's name `depositsCount`?

File: app/src/components/governance/ProposalDescription.vue [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
-          {{ description }}
-          <p class="parameter">New Parameter: {{ parameter }}</p>
+          <pre
+            v-if="
+              proposal.type === `PARAMETER_CHANGE` ||
+              proposal.type === `TREASURY`
+            "
+          >
+          {{ proposal.description }}
+          <p class="parameter">New Parameter: {{ proposal.changes }}</p>
```
Comment (faboweb) [inline app/src/components/governance/ProposalDescription.vue:14]: instead let's create the description on the server?

File: api/lib/source/polkadotV0-source.js [hunk 3]
```diff
@@ -807,26 +812,13 @@ class polkadotAPI {
         }
       })
     )
-    const votes = await Promise.all(
```
Comment (jbibla) [inline api/lib/source/polkadotV0-source.js:810]: why are `votes` being deleted?

File: app/src/ActionModal/components/ModalDeposit.vue (hunk 1 above)
Comment (jbibla) [reply app/src/ActionModal/components/ModalDeposit.vue:130]: what is this going to be used for?

File: app/src/components/governance/PageProposals.vue [hunk 4]
```diff
@@ -378,6 +378,7 @@ h4 {
   border-radius: 0.25rem;
   width: 100%;
   margin: 0 0.5rem;
+  white-space: nowrap;
```
Comment (jbibla) [inline app/src/components/governance/PageProposals.vue:381]: what does this look like on mobile?

File: api/lib/source/polkadotV0-source.js (hunk 3 above)
Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:810]: they were no votes, just deposits. This fixes the display and the logic in general

File: app/src/components/governance/ProposalDescription.vue (hunk 2 above)
Comment (Bitcoinera) [reply app/src/components/governance/ProposalDescription.vue:14]: ?? what do you mean?

File: app/src/components/governance/PageProposals.vue (hunk 4 above)
Comment (Bitcoinera) [reply app/src/components/governance/PageProposals.vue:381]: After another fix like this:

image

File: app/src/ActionModal/components/ModalDeposit.vue (hunk 1 above)
Comment (Bitcoinera) [reply app/src/ActionModal/components/ModalDeposit.vue:130]: I think I explained this already like 4 times :sweat_smile: but maybe Fabo wasn't there :shrug: 

the `numberOfSeconds` that you happily deleted, @faboweb , is needed for the api.tx.democracy.second extrinsic. And it is the deposit count, exactly. So we had the `depositTx` broken due to you PR. I can change the name, but this needs to be here

**Comment by faboweb** [comment]
unit tests failing

File: app/src/components/governance/ProposalDescription.vue (hunk 2 above)
Comment (faboweb) [reply app/src/components/governance/ProposalDescription.vue:14]: create the description on the server and not send "parameter" and "beneficiary"

File: app/src/ActionModal/components/ModalDeposit.vue (hunk 1 above)
Comment (faboweb) [reply app/src/ActionModal/components/ModalDeposit.vue]: I see. I am sorry, I didn't see that. For clarification: The tx needs to provide a maximum count of existing seconds. So people don't second if there are already n people seconding. Strange UX...

File: api/lib/source/polkadotV0-source.js (hunk 3 above)
Comment (jbibla) [reply api/lib/source/polkadotV0-source.js:810]: so the function name is no bueno: `getDemocracyProposalDetailedVotes`

**Pull request merged**